src/states/common/base.rs

```diff
@@ -99,6 +115,21 @@ pub trait Base: Display {
         Err(InterfaceError::InvalidState)
     }
 
+    fn handle_send_msg_to_client(
+        &mut self,
+        _peer_addr: SocketAddr,
+        _msg: NetworkBytes,
+        _token: Token,
+    ) -> Result<(), InterfaceError> {
+        warn!("{} - Cannot send message to client- invalid state.", self);
+        Err(InterfaceError::InvalidState)
+    }
+
+    fn handle_disconnect_client(&mut self, _peer_addr: SocketAddr) -> Result<(), InterfaceError> {
+        warn!("{} - Cannot handle DisconnectClient - invalid state.", self);
+        Err(InterfaceError::InvalidState)
```

Comment:
Same here (see https://github.com/maidsafe/routing/pull/1838/files#r333556207)

Reply:
Same reply as there :)

Reply:
Note this is also allowed for `adults` as you would want to disconnect if there was a demotion.